src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.civil.utils;
+
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CaseDataContentConverter {
+
+    private CaseDataContentConverter(){
+
+    }
+
+    public static CaseDataContent caseDataContentFromStartEventResponse(
+        StartEventResponse startEventResponse, Map<String, Object> contentModified) {
```

Comment:
With public static methods, maybe add some Javadoc expressing this method does, as it'll be available to other users/projects?

Reply:
i think it is already states what it is doing and it is already used in the similar service that does events for case workers.